web/modules/custom/dpl_fbi/dpl_fbi.install

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Drupal\Core\Database\Database;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use function Safe\sprintf as sprintf;
+
+/**
+ * Adds 'material_type' column to 'dpl_fbi_work_id' field storage.
+ *
+ * This update function alters the schema for custom field storage
+ * to include a new column for material type. It handles both the
+ * base table and the revision table for the field.
+ *
+ * The function is taken from drupal stackexchange:
+ * https://drupal.stackexchange.com/questions/255495/add-column-to-custom-field-schema
+ *
+ * @throws \Exception
+ *   Throws exception if the field type cannot be found or if the expected
+ *   field storage is not an instance of FieldStorageDefinitionInterface.
+ */
+function dpl_fbi_update_10201() :void {
```

Comment:
I wonder why you picked this version name for the update hook.

Reply:
I thought the standard was to use the drupal version you are on. So this would be drupal 10.2.x. 
I realise didn't check that we are actually on 10.1.7 now, which would make the hook 10101. 

If this isn't correct though let me know! 👀  🚀

Reply:
It is not. You can read more about the naming scheme here: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_N/10